src/modules/media/const/index.tsx

```diff
@@ -51,44 +51,44 @@ export const OBJECT_DETAIL_TABS = (mediaType: MediaTypes): TabProps[] => [
 export const MEDIA_ACTIONS = (): DynamicActionMenuProps => ({
 	actions: [
 		{
-			label: 'Quotes',
+			label: i18n?.t('modules/media/const/index___quotes') ?? '',
```

Comment:
If a translation key doesn't exist, then i18n will just output the key itself as the translation. So your `?? ''` will not be executed

Reply:
Deze conditional  is er enkel voor als `i18n` undefined is.

![image](https://user-images.githubusercontent.com/85545258/158959877-e35fcee7-9c04-4fcf-ab84-a2291bf23136.png)